README.md

```diff
@@ -11,16 +17,21 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
 
 ## Features
 
- - [Dev environments as code](https://www.gitpod.io/docs/#-dev-environments-as-code)
- - [Prebuilt dev environments](https://www.gitpod.io/docs/#prebuilds) - automatically prepare environments on every Git push
- - Professional developer experience in a browser tab (VS Code Extensions, full Linux terminals)
- - Integrated Docker build
- - GitLab, GitHub, and Bitbucket integration
- - Integrated code reviews
- - Snapshots - Snapshot any state of your dev environment and let others create clones
- - Collaboration - Invite team members to your environments
- - Gitpod CLI to automate your experience
- - [learn more](https://www.gitpod.io/features/)
+🏗 [Dev environments as code](https://www.gitpod.io/docs/#-dev-environments-as-code) - Gitpod applies lessons learned from infrastructure-as-code. Spinning up dev environments is easily repeatable and reproducible empowering you to automate, version-control and share dev environments across your team.
+
+⚡️ [Prebuilt dev environments](https://www.gitpod.io/docs/#prebuilds) - Gitpod continuously builds all your git branches like a CI server. Whenever new commits are pushed to your repository, Gitpod can prebuild dev environments, i.e. run the init commands in your `.gitpod.yml` before you even start a workspace.
```

Comment:
Gitpod continuously prebuilds dev environments for all you git branches similar to a CI server. Control how Gitpod prepares environments through `init` commands in your `.gitpod.yml`.

Reply:
i.e. Gitpod doesn't build your branches.